installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc

```diff
@@ -23,6 +23,15 @@ include::modules/understanding-agent-install.adoc[leveloffset=+1]
 
 * xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#installation-requirements-platform-none_preparing-to-install-with-agent-based-installer[Requirements for a cluster using the platform "none" option]
 
+* xref:../installing_bare_metal_ipi/ipi-install-prerequisites.adoc#network-requirements-increase-mtu_ipi-install-prerequisites[Increase the network MTU]
+
+* xref:../../nodes/nodes/nodes-sno-worker-nodes.adoc#nodes-sno-worker-nodes[Adding worker nodes to single-node OpenShift clusters].
++
+[NOTE]
+====
+To add worker nodes, you must have access to the OpenShift Cluster Manager. This method is not supported when using the Agent-based installer to install a cluster in a disconnected environment.
+====
+
```

Comment:
This note needs to be relocated.

Additional resources items must contain minimal-length, human-readable text between the square brackets, and no text outside of the square brackets. You can review the guidelines here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

Reply:
Got it. Let me relocate the Note.

Reply:
I have added the notehere in ["Adding worker nodes to SNO clusters"](https://67336--docspreview.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-sno-worker-nodes#nodes-sno-worker-nodes) module. @dfitzmau Can you check if it is right?